data/templated/example.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
<!-- Note: these example templates reuse TPC-C queries and data as a demonstration, but others are possible. -->